conda/conda-build/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "legate-core" %}
+{% if package_version is defined %}
+    {% set version = package_version %}
+# The tag must be in one of the two formats (where N is a digit):
+# vNN.NN.NN
+# vNN.NN.NN.dev
+{% elif 'dev' in environ.get('GIT_DESCRIBE_TAG', '') %}
+    {% set version = environ.get('GIT_DESCRIBE_TAG', '') ~ environ.get('GIT_DESCRIBE_NUMBER', '') %}
```

Comment:
Remember to strip the preceding "v" from the tag so that the version number is valid